common/src/main/java/xyz/wagyourtail/jsmacros/client/mixins/access/MixinPhaseType.java

```diff
@@ -0,0 +1,18 @@
+package xyz.wagyourtail.jsmacros.client.mixins.access;
+
+import net.minecraft.entity.boss.dragon.phase.PhaseType;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+/**
+ * @author Etheradon
+ * @since 1.8.4
+ */
+@Mixin(PhaseType.class)
+public interface MixinPhaseType {
+
+    @Accessor
+    String getName();
```

Comment:
is this even sync'd to the client by the server?

Reply:
The phase is synced to the client for animation purposes. There is even a `clientTick` method in the Phase class.